installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.adoc

```diff
@@ -29,6 +29,7 @@ Because the installation media is on the mirror host, you can use that computer
 ** Has firewall rules or a peering connection to access the mirror registry hosted elsewhere
 * If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to. While you might need to grant access to more sites, you must grant access to `*.googleapis.com` and `accounts.google.com`.
 * If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can xref:../../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[manually create and maintain IAM credentials].
+* You xref:../../installing/installing_gcp/installing-gcp-account.adoc#installing-gcp-account[configured a GCP project] to host the cluster.
```

Comment:
Better move the new prereq to the 3rd place, i.e. after "You read the documentation on..."

Reply:
Updated